<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Youngs-MacBook-Pro.local" name="main.java.com.dbyl.appiumCore.tests.iOSTest" tests="1" failures="0" timestamp="9 Apr 2016 11:53:11 GMT" time="0.004" errors="1">
  <testcase name="calc" time="0.004" classname="main.java.com.dbyl.appiumCore.tests.iOSTest">
    <error type="org.openqa.selenium.WebDriverException" message="An unknown server-side error occurred while processing the command. Original error: Could not find a device to launch. You requested &#039;iPhone 6 (9.3) [&#039;, but the available devices were: [&quot;iPad 2 (9.3) [F8547F97-8884-4E4E-822C-CB9EF616B0F8] (Simulator)&quot;,&quot;iPad Air 2 (9.3) [462B5685-E04E-48FA-852A-12ED106E7BA8] (Simulator)&quot;,&quot;iPad Pro (9.3) [F0C86AE1-6199-449C-AE51-FD7080177245] (Simulator)&quot;,&quot;iPad Retina (9.3) [B739863C-6213-442F-8B7A-2072CC00C758] (Simulator)&quot;,&quot;iPhone 5s (9.3) [05002BEB-0675-4F64-96F8-191449726FE8] (Simulator)&quot;,&quot;iPhone 6 Plus (9.3) [37A4FBB6-C3C1-482D-95BB-D3F1BE72C098] (Simulator)&quot;,&quot;iPhone 6s (9.3) [C4719D24-FABF-450E-B702-31CCDF8C4BB1] (Simulator)&quot;,&quot;iPhone 6s (9.3) + Apple Watch - 38mm (2.2) [1CC520F4-E4C1-4227-A7EC-EABE73518810] (Simulator)&quot;,&quot;iPhone 6s Plus (9.3) [396203D4-1C7D-45A5-B34C-E4926A321FF8] (Simulator)&quot;,&quot;iPhone 6s Plus (9.3) + Apple Watch - 42mm (2.2) [C2C81009-303F-46E3-BDE4-A64EC4A8AA4B] (Simulator)&quot;] (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 1.43 seconds
Build info: version: &#039;2.53.0&#039;, revision: &#039;35ae25b&#039;, time: &#039;2016-03-15 17:00:58&#039;
System info: host: &#039;Youngs-MacBook-Pro.local&#039;, ip: &#039;10.207.218.184&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.11.4&#039;, java.version: &#039;1.7.0_79&#039;
Driver info: io.appium.java_client.ios.IOSDriver">
      <![CDATA[org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: Could not find a device to launch. You requested 'iPhone 6 (9.3) [', but the available devices were: ["iPad 2 (9.3) [F8547F97-8884-4E4E-822C-CB9EF616B0F8] (Simulator)","iPad Air 2 (9.3) [462B5685-E04E-48FA-852A-12ED106E7BA8] (Simulator)","iPad Pro (9.3) [F0C86AE1-6199-449C-AE51-FD7080177245] (Simulator)","iPad Retina (9.3) [B739863C-6213-442F-8B7A-2072CC00C758] (Simulator)","iPhone 5s (9.3) [05002BEB-0675-4F64-96F8-191449726FE8] (Simulator)","iPhone 6 Plus (9.3) [37A4FBB6-C3C1-482D-95BB-D3F1BE72C098] (Simulator)","iPhone 6s (9.3) [C4719D24-FABF-450E-B702-31CCDF8C4BB1] (Simulator)","iPhone 6s (9.3) + Apple Watch - 38mm (2.2) [1CC520F4-E4C1-4227-A7EC-EABE73518810] (Simulator)","iPhone 6s Plus (9.3) [396203D4-1C7D-45A5-B34C-E4926A321FF8] (Simulator)","iPhone 6s Plus (9.3) + Apple Watch - 42mm (2.2) [C2C81009-303F-46E3-BDE4-A64EC4A8AA4B] (Simulator)"] (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 1.43 seconds
Build info: version: '2.53.0', revision: '35ae25b', time: '2016-03-15 17:00:58'
System info: host: 'Youngs-MacBook-Pro.local', ip: '10.207.218.184', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.4', java.version: '1.7.0_79'
Driver info: io.appium.java_client.ios.IOSDriver
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:249)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:144)
	at io.appium.java_client.DefaultGenericMobileDriver.<init>(DefaultGenericMobileDriver.java:37)
	at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:162)
	at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:171)
	at io.appium.java_client.ios.IOSDriver.<init>(IOSDriver.java:56)
	at main.java.com.dbyl.appiumCore.tests.iOSTest.setUp(iOSTest.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:216)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:590)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:75)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:46)
]]>
    </error>
  </testcase> <!-- calc -->
</testsuite> <!-- main.java.com.dbyl.appiumCore.tests.iOSTest -->
